src/order/atoms.lean

```diff
@@ -88,6 +107,76 @@ is_simple_lattice_iff_is_simple_lattice_order_dual.1 (by apply_instance)
 
 end is_simple_lattice
 
+namespace is_simple_lattice
+
+variables [bounded_lattice α] [is_simple_lattice α]
+
+/-- A simple `bounded_lattice` is also distributive. -/
+protected def bounded_distrib_lattice : bounded_distrib_lattice α :=
+{ le_sup_inf := λ x y z, by { rcases eq_bot_or_eq_top x with rfl | rfl; simp },
+  .. (infer_instance : bounded_lattice α) }
+
+section decidable_eq
+variable [decidable_eq α]
+
+@[priority 200]
+instance : fintype α :=
+{ elems := {⊥, ⊤},
+  complete := λ x, finset.mem_insert.2 (or.imp_right finset.mem_singleton.2 (eq_bot_or_eq_top x)) }
```

Comment:
Please make it defeq [bool.fintype](https://leanprover-community.github.io/mathlib_docs/data/fintype/basic.html#bool.fintype)

Reply:
I think I've achieved this.

Reply:
`example : (is_simple_lattice.fintype : fintype bool) = bool.fintype := rfl` works.